migrations/versions/2023_02_17_0426-981a17a06c13_create_posts_table.py

```diff
@@ -0,0 +1,45 @@
+"""create posts table
+
+Revision ID: 981a17a06c13
+Revises: e8634796ad93
+Create Date: 2023-02-17 04:26:45.267109+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '981a17a06c13'
+down_revision = 'e8634796ad93'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('posts',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('author_id', sa.Integer(), nullable=False),
+    sa.Column('created_at', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+    sa.Column('body', sa.String(length=123), nullable=False),
+    sa.Column('topic_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['author_id'], ['users.id'], ),
+    sa.ForeignKeyConstraint(['topic_id'], ['topics.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.alter_column('topics', 'created_at',
```

Comment:
Seems like you haven't changed this column, therefore this line shouldn't be generated. Did you change this migration manually?

I think it will be better to regenerate it.